.changeset/tall-bobcats-trade.md

```diff
@@ -0,0 +1,34 @@
+---
+'@xstate/react': patch
+'@xstate/store': patch
```

Comment:
To be fair - this change doesn't really affect `@xstate/store` so it feels off to me that it's going to be mentioned in its CHANGELOG cause of this changeset. The change itself is solely in `@xstate/react`

Reply:
```suggestion
```